README.rst

```diff
@@ -5,18 +5,23 @@ Client interface for Scrapinghub API
 .. image:: https://secure.travis-ci.org/scrapinghub/python-scrapinghub.png?branch=master
    :target: http://travis-ci.org/scrapinghub/python-scrapinghub
 
+
+The ``scrapinghub`` is a Python library for communicating with the `Scrapinghub API`_.
+
+
+.. contents:: :depth: 1
+
+
 Requirements
 ============
 
 * Python 2.6 or above
 * `Requests`_ library
```

Comment:
as well as 'retrying>=1.3.3', 'six>=1.10.0'. we either need to keep this section up to date with setup.py or remove it (e.g. I don't see such section in https://github.com/kennethreitz/requests or )

what is missing is Installation section with `pip install python-scrapinghub`, + mention that msgpack support can be enabled by installing `pip install python-scrapinghub[msgpack]`.